src/ImageSharp/PixelFormats/Rgba32.cs

```diff
@@ -220,6 +220,32 @@ public uint PackedValue
             set => this.Rgba = value;
         }
 
+        /// <summary>
+        /// Gets the component value at the given index
+        /// </summary>
+        /// <param name="index">The component index</param>
+        /// <returns>The <see cref="byte"/></returns>
+        public byte this[int index]
```

Comment:
We need a more consistent behavior if we want to expose this publicly. Currently it:
- Throws in debug mode for an index being out of range (indicating an incorrect usage)
- Silently, returns Alpha in release mode (this behavior is undocumented!)

Reply:
I'll refactor to not use `default` and throw after the switch

Reply:
do we event need this at all? its only used here https://github.com/SixLabors/ImageSharp/blob/65b419085131fd6b7ba32d2709d0ce3bb61a4d3b/src/ImageSharp/Dithering/Ordered/OrderedDitherBase.cs#L29-L34 

I feel its probably better to just moving the switch inline to that method and not add it unnecessarily to the public api